src/templates/admin/repository/author_article.html

```diff
@@ -146,6 +146,27 @@ <h2>Dates</h2>
                         {% endfor %}
                     </table>
                 </div>
+                <div class="title-area">
+                    <h2>Metrics</h2>
+                </div>
+                <div class="content">
+                    <p>Views track how often the article page has been visited, while Downloads record the number of times a
```

Comment:
Curious as to why its a generic `the article page` but then `{{ request.repository.object_name }}'s ` for downloads.  

If that's because this is being displaced on the article page, then maybe use **this** article page rather than **the**.